subsys/bluetooth/host/smp.c

```diff
@@ -1632,7 +1632,15 @@ static void smp_pairing_complete(struct bt_smp *smp, uint8_t status)
 {
 	struct bt_conn *conn = smp->chan.chan.conn;
 
-	LOG_DBG("status 0x%x", status);
+	LOG_DBG("got status 0x%x", status);
```

Comment:
Isn't it some sort of local modification to help with the issue investigation that got included by accident?

```suggestion
	LOG_DBG("status 0x%x", status);
```

Reply:
No I did that because after that message, the status value can be different from the one that got initially given to the function.

It's a small thing, I can revert if you prefer 🙂

Reply:
I see, no need to change that then